src/Commands/Command.php

```diff
@@ -104,4 +104,30 @@ public function setHidden($hidden)
 
         return $this;
     }
+
+    /**
+     * Prompt the user for input.
+     *
+     * @param  string  $question
+     * @param  string|null  $default
+     * @param  array|string|null  $validationRule
+     * @return mixed
+     */
+    public function ask($question, $default = null, $validationRule = null)
+    {
+        if (! class_exists(\Illuminate\Validation\Validator::class) || ! $this->app->bound('validator')) {
```

Comment:
Should this always be called? 🤔 I was thinking that perhaps it would be easier to only check for the class existing and being bound if a validation rule is provided. Otherwise won't existing applications that use `ask()` break?

Reply:
You're totally right, oversight on my part.